### DISCUSSION TITLE
Feature request: ability to aggregate multiple dashboards into a single 'overview' dashboard

### DISCUSSION THREAD

**Discussion by tstibbs** [grafana/grafana#71125] [Dashboards, Panels & Visualizations]
I have outlined a specific use-case below to explain why I'd like this functionality. My use-case could be achieved in a variety of ways, however one simple way might be to take the current functionality which allows sharing of queries between panels _within_ a dashboard, and extend it to work _across_ dashboards. This might be a more useful generic feature however I have outlined my specific use-case for context anyway.

## Situation ##

I have multiple environments (10 ish), with a dashboard per environment.

## Requirement ##

I would like to create a centralised 'overview 'dashboard which aggregates the contents of the others. E.g. the the centralised dashboard would show a single stat visualisation corresponding to each of the environment-specific dashboards, which would turn red if anything on the sub dashboard tripped the thresholds. You could then click through to view the environment-specific dashboard and dig deeper into what the specific problem was.

## Rationale ##

Having a single dashboard per environment is sensible - it means the team looking after that environment can focus on just the stuff they need and means we're only running the queries we need to.

Having an overview dashboard is useful for management, to see which environments have problems and thus for example ensure staff are assigned appropriately. They don't want to have to look through all of the environment-specific dashboards.

## Problem ##

The only way I can see to achieve this is to create the centralised dashboard as a totally separate thing, completely unconnected to any of the environment-specific dashboards. I could hand-craft queries and transformations which pull all the same data as the environment-specific dashboards.

However, this adds a maintenance burden - any changes in the queries in the sub dashboards would not automatically be reflected in the overview dashboard, and copying changes across would likely be complex because of the sheer size and complexity of the queries in the overview dashboard. Changes to thresholds would be even more complex, because they're likely configured through the thresholds UI in the sub dashboard but would likely have to be encoded into expressions/transforms in the overview dashboard. I **really** don't want to have to configure everything up in two places just to be able to see an overview. Grafana is about bringing all your data together in one place and giving an overview right?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82601. Feel free to continue the discussion around this there. Thank you!
